list-navigation/lambda/handlers/select-item.ts

```diff
@@ -38,18 +38,21 @@ export class SelectItemHandler extends BaseApiHandler {
         super(apiName);
     }
 
-    handle(handlerInput : HandlerInput): Response {
+    async handle(handlerInput : HandlerInput){
```

Comment:
just for clarity sake, can we keep the defintion of the what the return type is? should be `Promise<Response>` now?

Reply:
Noted.